holoviews/plotting/bokeh/element.py

```diff
@@ -1832,7 +1838,20 @@ def _get_color_data(self, element, ranges, style, name='color', factors=None, co
 
     def _get_cmapper_opts(self, low, high, factors, colors):
         if factors is None:
-            colormapper = LogColorMapper if self.logz else LinearColorMapper
+            if self.logz:
+                colormapper = LogColorMapper
+                if util.is_int(low) and util.is_int(high) and low == 0:
+                    low = 1
+                    if 'min' not in colors:
+                        colors['min'] = 'rgba(0, 0, 0, 0)'
```

Comment:
At first I thought this would be a black color, which wouldn't always be appropriate, but now I see that alpha is zero as well, so it's transparent.  Maybe add a comment to clarify:

```suggestion
                        # Make integer 0 be transparent
                        colors['min'] = 'rgba(0, 0, 0, 0)'
```